lib/cfn-nag/cfn_nag.rb

```diff
@@ -82,8 +82,8 @@ def audit(cloudformation_string:, parameter_values_string: nil)
                                       parameter_values_string
       violations += @custom_rule_loader.execute_custom_rules(cfn_model)
       violations = filter_violations_by_profile violations
-    rescue Psych::SyntaxError, ParserError => parser_error
-      violations << fatal_violation(parser_error.to_s)
+    rescue Psych::SyntaxError, ParserError => exception
```

Comment:
what is the value in changing this to have a less specific name?

Reply:
I think it is attempting to force consistency.